Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -3002,6 +3002,15 @@ private static bool HandleKeywords(Material destMat, string keyword)
         m_tmpV3.Dispose();
         m_tmpV4.Dispose();
         m_tmpC.Dispose();
+        
+#if UNITY_EDITOR
```

Comment:
Why is it `UNITY_EDITOR` only ?

Reply:
The rendertexture lifetime is different at runtime. In editor we save it to a texture and release it but didn't release them when the server was destroyed without saving.
I didn't encounter this at runtime, I'm not sure if it's safe to release them there.